SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -17,13 +17,17 @@
                 "bmake",
                 "brotli",
                 "busybox",
+                "byacc",
                 "calamares",
                 "catch",
                 "chrony",
                 "conntrack-tools",
                 "cpprest",
                 "cryptsetup",
+                "CUnit",
+                "Cython",
```

Comment:
Don't add cython here since you're not modifying it.

Reply:
Done